storage/aws/s3/variables.tf

```
@@ -103,3 +103,9 @@ variable "object_storage_adapter" {
   type        = string
   default     = "ArmoniK.Adapters.S3.ObjectStorage"
 }
+
+variable "service_account_role_name" {
```

Service account is a Kubernetes concept, but here, we are dealing with the AWS object which is the role.

```suggestion
variable "role_name" {
```